internal/database/migration/schemas/description.go

```diff
@@ -252,7 +252,14 @@ func (d ColumnDescription) AlterToTarget(table TableDescription, target ColumnDe
 		d.IsNullable = target.IsNullable
 	}
 	if d.Default != target.Default {
-		statements = append(statements, fmt.Sprintf("ALTER TABLE %s ALTER COLUMN %s SET DEFAULT %s;", table.Name, target.Name, target.Default))
+		var verb string
+		if target.Default == "" {
+			verb = "DROP"
+			statements = append(statements, fmt.Sprintf("ALTER TABLE %s ALTER COLUMN %s %s DEFAULT;", table.Name, target.Name, verb))
+		} else {
+			verb = "SET"
+			statements = append(statements, fmt.Sprintf("ALTER TABLE %s ALTER COLUMN %s %s DEFAULT %s;", table.Name, target.Name, verb, target.Default))
+		}
```

Comment:
```suggestion
		if target.Default == "" {
			statements = append(statements, fmt.Sprintf("ALTER TABLE %s ALTER COLUMN %s DROP DEFAULT;", table.Name, target.Name))
		} else {
			statements = append(statements, fmt.Sprintf("ALTER TABLE %s ALTER COLUMN %s SET DEFAULT %s;", table.Name, target.Name, target.Default))
		}
```

You don't need to declare the outer-scope variable as you only assign it in the if consequent that you use it in.